src/iuse.cpp

```diff
@@ -2339,21 +2169,30 @@ int iuse::radio_mod( player *p, item *, bool, const tripoint& )
 
 int iuse::remove_all_mods(player *p, item *, bool, const tripoint& )
 {
-    static const std::vector<std::string> removable_mods = {{
-        "DOUBLE_AMMO", "USE_UPS", "ATOMIC_AMMO"
-    }};
+    if( !p ) {
```

Comment:
`remove_item` could remove the item from the active item vectors. One of the reasons I want to use `item_location` is that it makes vehicle mounted tools or interaction with static furniture much easier.